roles/ceph-nfs/templates/ganesha.conf.j2

```diff
@@ -52,6 +52,13 @@ EXPORT
         {{ ganesha_rgw_export_overrides | default(None) }}
 
 }
+
+RGW {
+        ceph_conf = "/etc/ceph/{{ cluster }}.conf";
+        cluster = "{{ cluster }}";
+        name = "{{ rgw_client_name }}";
```

Comment:
@leseb name here is the same as "client.rgw.nfs0" here: https://paste.fedoraproject.org/paste/44GgmshCXkxr~zWoiS2VEg